pkg/controller/provider/validation.go

```diff
@@ -258,7 +258,6 @@ func (r *Reconciler) validateSecret(provider *api.Provider) (secret *core.Secret
 	case api.Ova:
 		keyList = []string{
 			"url",
-			"insecureSkipVerify",
```

Comment:
Its really not needed here, can be drooped. The mutator will add this value any way, so if we want to entirely remove this value from the secret we need to exclude ova from there as well.

Reply:
following the other PR comment we can entirely drop it.